tests/flytekit/unit/common_tests/types/test_schema.py

```diff
@@ -52,3 +52,20 @@ def test_typed_schema():
         assert len(b.type.columns) == len(_ALL_COLUMN_TYPES)
         assert list(b.type.sdk_columns.items()) == _ALL_COLUMN_TYPES
         assert b.remote_location.startswith(t.name)
+
+
+# Ensures that subclassing types works inside a schema.
+def test_casting():
+    class MyDateTime(primitives.Datetime):
+        ...
```

Comment:
this works? `...` is an allowed syntax?

Reply:
after i disabled travis's 2.7 tests, yup.